### DISCUSSION TITLE
Compiling Error

### DISCUSSION THREAD

**Discussion by SpongeSause** [KadeArchive/Kade-Engine#483] [Q&A] [answered]
Hi, so I'm pretty new to coding and stuff so I tried to follow the guides here as much as I could, so I got everything downloaded and made a few tiny edits in StoryMenuState.hx and tried to build to html5 with the code "lime test html5 -debug" and got this error once finished and I have no clue where to go from here. Btw I'm on Windows if that helps at all.

![error](https://user-images.githubusercontent.com/84270053/118393770-bf499500-b60e-11eb-9010-a71e6866c6a7.png)

**Comment by puyoxyz** [comment, marked as answer]
html5, linux and mac builds are broken on 1.4.2. use the latest source code instead
